### DISCUSSION TITLE
Make Client Library Order Configurable

### DISCUSSION THREAD

**Discussion by wolveix** [scalar/scalar#976] [Ideas]
Hey! Furthering the discussion on Discord, it'd be fantastic if we could configure the order of the client libraries displayed at the top of the spec :)

**Comment by hanspagel** [comment]
Oh yeah, make sure to subscribe to this issue:

https://github.com/scalar/scalar/issues/930

**Reply by wolveix** [reply]
Apologies, Marc suggested creating this yesterday so I assumed (and therefore didn't search for existing suggestions) that it hadn't already been suggested 😅
